libs/application/templates/financial-aid/src/forms/Submitted.ts

```diff
@@ -14,12 +14,13 @@ export const Submitted: Form = buildForm({
   mode: FormModes.APPLYING,
   children: [
     buildSection({
-      id: 'Stuff TODO',
-      title: 'TODO stuff',
+      id: 'submitted',
+      title: m.section.received,
       children: [
-        buildTextField({
-          id: 'Todo-text-field',
-          title: 'Submitted',
+        buildCustomField({
+          id: 'residenceChangeConfirmation',
+          title: m.confirmation.general.pageTitle,
+          component: 'Confirmation',
```

Comment:
Just wondering, am I understanding it correctly that this will later be replaced with the status page component?

Reply:
Well, yes and no, when you submit an application you will go to this site to get confirmation that the application has successfully been submitted. However when I do the check in the beginning of the flow than I send the user here if he has applied for this current period, so in that scenario it is a placeholder